src/libs/actions/Policy.ts

```diff
@@ -425,7 +425,6 @@ function setWorkspaceAutoReporting(policyID: string, enabled: boolean) {
             key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
             value: {
                 autoReporting: !enabled,
-                autoReportingFrequency: null,
```

Comment:
I can't comment on line 416, but it seems like we just wanna set it to weekly if the value is `null` or `Instant`, right? Otherwise we just want to use the previous frequency we had set